src/main/cpp/Autonomous.cpp

```diff
@@ -909,22 +908,26 @@ double Autonomous::gettime_d(){
 
 bool Autonomous::TurnToAngle(float angle, bool positive) { // rotates bot in place to specific angle
    
-    if(!positive){
-        speed = -1*speed;
-    }
-    //if (fabs(a_Gyro->getAngle() - angle) >= 5) {
-    //if((a_Gyro->getAngle() - angle <= 10) || ((a_Gyro->getAngle() - angle - 360) <= 10) || (a_Gyro->getAngle() - (angle + 360) <= 10)){
-       if(fabs(a_Gyro->getAngleClamped() - angle) <= 5){
+//     if(!positive){
+//         speed = -1*speed;
+//     }
+    if (a_SwerveDrive->turnToAngle(angle, positive)) {
+//     //if((a_Gyro->getAngle() - angle <= 10) || ((a_Gyro->getAngle() - angle - 360) <= 10) || (a_Gyro->getAngle() - (angle + 360) <= 10)){
+//        if(fabs(a_Gyro->getAngleClamped() - angle) <= 5){
         a_SwerveDrive->stop();
         return true;
-       
-
-}
-    else {
-        a_SwerveDrive->turnToAngle(angle, positive);
+    }
+    else{
         return false;
     }
+       
+
 }
+    // else {
+    //     a_SwerveDrive->turnToAngle(angle, positive);
+    //     return false;
+    // }
```

Comment:
Probably better to go ahead and delete the code that's commented out in this function if you're thinking we don't need it any more. We do always have the history of this file to look back on if we need to reference it later.